devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -84,7 +84,7 @@ def parse(String description) {
     log.debug "description:- ${description}"
     if (description?.startsWith("read attr -")) {
         Map descMap = zigbee.parseDescriptionAsMap(description)
-        if (descMap?.clusterInt == CLUSTER_WINDOW_COVERING && descMap.value) {
+        if (usesWindowCovering() && descMap?.clusterInt == CLUSTER_WINDOW_COVERING && descMap.value) {
```

Comment:
@tpmanley Do you think we should maybe not check here for usesWindowCovering and just handle the messages that we receive from the device?

Reply:
I think that's OK to handle everything as long as one device doesn't actually use both and they either provide conflict information or both report at the exact same time and lead to the possibility of duplicate events

Reply:
Does the Wistar report ATTRIBUTE_POSITION_LIFT but it's not correct or not report it at all? If not at all then I think you can remove `usesWindowCovering` and just handle every message that is received. If it reports the value and it's wrong so it needs to be ignored then I think you can replace `usesWindowCovering` with `supportsLiftPercentage` and then delete `usesWindowCovering`

Reply:
@BYDengS ^

Reply:
@tpmanley  No, it not report it at all, I will delete it.